utils/mvn/utils.go

```diff
@@ -70,8 +71,9 @@ func getMavenHomeAndValidateVersion() (string, error) {
 				return "", err
 			}
 		}
```

Comment:
Looking at the following code - 
```
	// Finding the relevant "Maven home" line in command response.
	for _, line := range output {
		if mvnHome == "" && strings.HasPrefix(line, "Maven home:") {
			// The M2_HOME environment variable is not defined.
			// Since Maven installation can be located in different locations,
			// Depending on the installation type and the OS (for example: For Mac with brew install: /usr/local/Cellar/maven/{version}/libexec or Ubuntu with debian: /usr/share/maven),
			mvnHome, err = parseMvnHome(line)
			if err != nil {
				return "", err
			}
		}
		if strings.Contains(line, "Apache Maven") {
			versionRegex := regexp.MustCompile("Apache\\sMaven\\s(\\w[\\w-\\.]+)\\s")
			mvnVersion = versionRegex.FindStringSubmatch(line)[1]
		}
	}
```
shouldn't we break out of the loop as soon as ```mvnHome``` is not empty, right after - 
```
			mvnHome, err = parseMvnHome(line)
			if err != nil {
				return "", err
			}
```
And fo so, there's no need to check the value of mvnHome in the following if statement.
```
if mvnHome == "" && strings.HasPrefix(line, "Maven home:") {
```

In addition, the following line at the beginning of the function may set a value into the ```mvnHome``` variable - 
```
mvnHome := os.Getenv(mavenHome)
```
If mvnHome include a value, we probably shouldn't get into the loop to try and set it.